browser/ui/views/infobars/brave_infobar_container_view.cc

```diff
@@ -14,7 +14,7 @@ BraveInfoBarContainerView::BraveInfoBarContainerView(
     : InfoBarContainerView(delegate) {
   // To hide shadow, replace it with empty view.
   DCHECK(content_shadow_);
-  RemoveChildViewT(content_shadow_);
+  RemoveChildViewT(content_shadow_.ExtractAsDangling());
```

Comment:
curious why `content_shadow_ ` is dangled. it's assigned again in the next line.

Reply:
The pointer will be pointing to something that has been destroyed during the remove child call by the time it reaches the next line, at which point it detects it is pointing to invalid memory, as the reassignment is taking place.